_data/meltano/loaders/target-mongodb/hotgluexyz.yml

```
@@ -0,0 +1,28 @@
+name: target-mongodb
+variant: hotgluexyz
+label: MongoDB
+logo_url: /assets/logos/loaders/mongodb.png
+capabilities:
+- about
+description: MongoDB
```

```suggestion
description: General purpose, document-based, distributed database.
```
So it matches the tap.